public/employers/arcadia.yml

```diff
@@ -0,0 +1,39 @@
+name: Arcadia Group Ltd.
+aliases:
+  - Burton
+  - Dorothy Perkins
+  - Evans
+  - Miss Selfridge
+  - Outfit
+  - Topman
+  - Topshop
+  - Wallis
+shortName: Arcadia
+wiki: Arcadia Group
+image: arcadia.svg
+officialWebsite: https://www.arcadiagroup.co.uk/about-arcadia/history
+location:
+  city: London
+  country: gb
+  international: true
+  wiki: London
+employeesBefore:
+  type: approximately
+  upperBound: 22000
+  year: 2019
+summary: After furloughing over 14,500 employees, most of them shop workers, Arcadia Group has requested and used taxpayer money to pay wages instead of using money from its own accounts and wealthy executives.
+citations:
+  - summary: Multibillionaire Philip Green, the chairman of Arcadia Group and director of Arcadia's owner, Taveta Investments, has requested and used taxpayer funds to help pay employee wages after furloughing 14,500 workers.
+    positivity: -1
+    type: publication
+    sources:
+      - name: Daily Mail UK
+        link: https://web.archive.org/web/20200403135504/https://www.dailymail.co.uk/news/article-8183763/Sell-one-yachts-Outrage-billionaire-Sir-Philip-Green-asks-taxpayer-help.html
```

Comment:
Any reason why you're linking the web archive link?

Reply:
Honestly, we should do this whenever possible, so that the data stays usable no matter when in the future people look at it.

I'd do it for the second one, but there is no web.archive link for it.